style.css

```diff
@@ -0,0 +1,44 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
+html {
+
+}
```

Comment:
It is bad practice to have an empty rule-set. I suggest you add some property to the rule set or you remove it if it is not needed.